### PR TITLE
884 accordion styling(normal&dark theme)

### PR DESCRIPTION
**Pull request by Pooja-kj**
[Chore] Accordion Styling (Normal & Dark theme) #884

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Style Update (CSS)
- [ ] Other... Please describe:

Resolves 884

## What is the new behavior?
Changed below styles
Default Accordion

Header should be equivalent to an H4 (18px - 1.125rem).
The body contains unexpected padding around the content (1.5rem top/bottom) where UX outlined in as 1rem all around.
The body text should always be 14px (0.875rem) default.

Slim Accordion

Header should be equivalent to an H5 (14px - 0.875rem).
Header section padding should be 10px (0.625rem) top and bottom and 16px(1rem) left and right
The body text should always be 14px (0.875rem) default.
The padding around the body should be 16px (1rem) all around.

Dark Theme

Follows the above rules with the following additions
Header background color when collapsed is #313536
Header background color hovered when collapsed is #313536 3.5% darkened (HEX equivalent #292C2D)
Header background color when expanded is #181C1C.
Header background color hovered when expanded is #181C1C 3.5% darkened (HEX equivalent #101212)
border-radius needs to be 0px when the accordion reached the full width of the off-canvas container.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Saravanan-Tangoe — APPROVED** [review]
LGTM 👍🏻 , Thanks

File: projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.scss [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
     cursor: pointer;
     display: flex;
-    font-size: 1.2rem;
+    font-size: 1.125rem;
     padding: 1.2rem 1rem;
```
[the remark this answers is not in the document]
Comment (Pooja-kj) [reply projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.scss:71]: updated

File: projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.scss [hunk 2]
```diff
@@ -132,12 +135,12 @@
   }
 
   &__content {
-    padding: 1.5rem 1rem;
     font-size: 0.875rem;
+    padding: 1rem 1rem;
```
[the remark this answers is not in the document]
Comment (Pooja-kj) [reply projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.scss]: Done

**Comment by stolve** [comment]
@kpmurphh and myself have no way to provide approval/feedback for this issue. We do not have an instance of goponents on our systems in order to look at the visual fix and we are not in a position to give feedback on coding approaches. Please attach a screenshot of the update or block time on one of our calendars for a review session.

**Review by stolve — COMMENTED** [review]
I need to be provided with a screenshot or some visual to compare the issue to the fix.

**Pull request merged**